_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -118552,7 +118552,7 @@
 "edg" = (
 /obj/machinery/doorButtons/airlock_controller{
 	name = "Virology Access Console";
-	step_x = -18;
+	pixel_x = -18;
 	pixel_x = -8;
```

Comment:
Yeah its a bit difficult to concentrate properly in this 41 degree Celsius heat. Thanks for the encouragement.